models/organization/org_time/org_times_test.go

```diff
@@ -0,0 +1,305 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package orgtime
```

Comment:
Use `orgtime_test` to avoid dependencies cycle.

Reply:
Fixed!